packages/pass-style/src/error.js

```diff
@@ -24,34 +24,44 @@ const errorConstructors = new Map([
 export const getErrorConstructor = name => errorConstructors.get(name);
 harden(getErrorConstructor);
 
+const checkErrorLike = (candidate, check = undefined) => {
+  const reject = !!check && (details => check(false, details));
+  // TODO: Need a better test than instanceof
```

Comment:
Do you hope for a brand check?

This may be a good opportunity to add an issue link to the TODO comment.

Reply:
In @ljharb's meaning of the term, yes. The error stack proposal would let us do that. In fact, our partial shimming of that proposal may already do that on v8, firefox, and xs, depending on what their corresponding primitives do. Worth looking at! But not in this PR.

In this PR, the code within `checkErrorLike`, including this comment, was just moved from code within a method of the ErrorHelper. This PR is not trying to improve the semantics of any of that.

Reply:
Incidentally, pattern matching would also provide such a check for errors.